airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -298,39 +304,36 @@ def _collect_bulk_errors(self, response: requests.Response) -> List[Optional[dic
             )
 
     def _job_healthcheck(self, response: requests.Response) -> Optional[Exception]:
-        try:
-            # save the latest request to retry
-            self._save_latest_request(response)
-
-            # get the errors, if occured
-            errors = self._collect_bulk_errors(response)
+        # save the latest request to retry
```

Comment:
This is a fix requested by @bazarnov as there were some cases where we would retry too much

Reply:
Correct, looks good.